tests/Microsoft.Data.Analysis.Tests/BufferTests.cs

```diff
@@ -188,6 +188,7 @@ public void TestArrowStringColumnClone()
                 Assert.Null(clone[i]);
         }
 
+#if !NETFRAMEWORK // https://github.com/dotnet/corefxlab/issues/2796
```

Comment:
I'm curious, how did you discover this? We don't have CI running NETFramework right?

Reply:
With this PR we are now running tests against .NET Framework. See https://github.com/dotnet/corefxlab/pull/2797/files#diff-08cf3efcb8a6af2b5b7400915d600676R4